app/routes/circulars/circulars.lib.ts

```diff
@@ -17,6 +17,10 @@ export function bodyIsValid(body: string) {
   if (body.length) return true
 }
 
+export function emailIsAutoReply(subject: string) {
+  return emailAutoReplyChecklist.some((x) => subject.toLowerCase().includes(x))
```

Comment:
Factor `subject.toLowerCase()` outside of the anonymous function so that it is only evaluated once.